### ISSUE TITLE
FamilyParameter.GroupType node returns null instead of INVALID when inputting "Other" 

### ISSUE THREAD

**Issue by EATREVITPOOPCAD** [erfajo/OrchidForDynamo#245]
### Operating system
Windows 10

### Revit and Dynamo version
(_Which version of Revit, Dynamo and Orchid are you using? Go to Help > About if you're not sure. for Orchid version, go to Packages > Manage Packages_)
Revit: 2023.1.3
Dynamo: x.x.x.x
Orchid: x.x.x.x

### What did you do? 
FamilyParameter.GroupType node returns null instead of INVALID when inputting "Other" .  ( https://www.revitapidocs.com/2023/9942b791-2892-0658-303e-abf99675c5a6.htm "Other" is missing here too but it should be there for INVALID. Assuming Autodesk had this initially as a catch all but did not document that its used for "Other")

### What did you expect to see?
return grouptype enum INVALID

### What did you see instead?
null

![param groups](https://github.com/erfajo/OrchidForDynamo/assets/91353212/c21c1f47-29fd-487b-89c2-266d974eb556)

**Comment by erfajo** [comment]
You are right, the Dynamo team has somehow deprecated "INVALID", it is not the Revit team... but it is annoying. I don’t know what to do in writing about it, but I am considering writing my own forge units. For now, I can't help you :-S

**Comment by EATREVITPOOPCAD** [comment]
Shouldn't have been so quick to point the finger! 'Identity Data' it is then :D
